client/src/commons/AlphabetTextField/HebrewTextField.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import * as yup from 'yup';
+
+import AlphbetTextFieldType from './AlphabetTextFieldTypes';
```

Comment:
You should create HebrewTextFieldTypes.ts, also create a different folder in general for HebrewTextField